tests/test_bonded.py

```diff
@@ -178,7 +178,8 @@ def test_harmonic_bond(self, n_particles=64, n_bonds=35, dim=3):
 
             # test bitwise commutativity
             test_potential = potentials.HarmonicBond(bond_idxs, lamb_mult, lamb_offset)
-            test_potential_rev = potentials.HarmonicBond(bond_idxs[:, ::-1], lamb_mult, lamb_offset)
+            bond_idxs_rev = bond_idxs[:, ::-1] if n_bonds else bond_idxs
```

Comment:
Surprised the new conditional is needed, since

```python
>>> import numpy as np
>>> x = np.zeros(shape=(0, 2))
>>> (x == x[:, ::-1]).all()
True
```

Clarification: Is `bond_idxs.shape` not `(0, 2)` in the empty bonded terms case?

Reply:
It was, I updated in [c6ea7e9](https://github.com/proteneer/timemachine/pull/719/commits/c6ea7e9b6810816ba18f44a9b4b1f5f8eb95822b)